### PR TITLE
graphQl-784: added validation for the lowercase country id in the quote address

### PR DESCRIPTION
**Pull request by VitaliyBoyko**
### Description (*)
Added validation with the message "Country Code cannot contain lowercase characters."

### Fixed Issues (if relevant)
1. #784: Cover lowercase countryId in the shipping/billing address if provided by variables

### CODE REVIEW

**Comment by lenaorobei** [comment]
@magento run all tests

File: app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php [hunk 1]
```diff
@@ -66,7 +66,6 @@ public function execute(ContextInterface $context, CartInterface $cart, array $s
         }
 
         if (null === $customerAddressId) {
-            $addressInput['country_code'] = strtoupper($addressInput['country_code']);
```
Comment (lenaorobei) [inline app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php:69]: We should preserve this behaviour.

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php [hunk 2]
```diff
@@ -637,6 +637,50 @@ public function testSetNewBillingAddressWithRedundantStreetLine()
         $this->graphQlMutation($query, [], '', $this->getHeaderMap());
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/GraphQl/Catalog/_files/simple_product.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/customer/create_empty_cart.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/add_simple_product.php
+     * @expectedException \Exception
+     * @expectedExceptionMessage "Country Code" cannot contain lowercase characters.
```
Comment (lenaorobei) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php]: All tests with lowercase `country_code ` should pass.

File: app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php (hunk 1 above)
Comment (VitaliyBoyko) [reply app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php:69]: I will transfer this code to the factory. In this case, it works only for the shipping address.

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5494](https://jira.corp.magento.com/browse/ENGCOM-5494) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**